[inductor] Fix cuda compilation under fbcode remote execution

int3 opened the pull request:
Differential Revision: D57390072




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

chenyang78 commented on torch/_inductor/codecache.py:
```diff
@@ -2652,22 +2656,27 @@ def _cutlass_include_paths() -> List[str]:
+    ]
     extra_ldflags: List[str] = []
     if is_linux():
-        extra_lib_dir = "lib64"
```
Seems we are not including this extra lib dir anymore for all linux versions. Would this affect non-fbcode cases?

int3 replied:
Actually yeah. I realize that `_transform_cuda_paths_for_fbcode` actually handles this case too, we just need to call it unconditionally (and probably rename it to drop the 'fbcode' part)

int3 committed:
[inductor] Fix cuda compilation under fbcode remote execution (#126408)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/126408

Reviewed By: hl475

Differential Revision: D57390072
